src/pudl/transform/ferc1.py

```diff
@@ -6677,6 +6739,55 @@ def infer_intra_factoid_totals(
     return calcs_with_totals
 
 
+def add_calculation_component_corrections(
```

Comment:
me q: rn this is only being used to compile the big mutli-table calc component table. can i use this in the table transforms as well?